README.md

```diff
@@ -2,6 +2,9 @@
 
 This project was bootstrapped with [Nimble React template](https://github.com/nimblehq/react-templates).
 
+# Prerequisites
+nodejs latest
```

Comment:
I think it's better to specify the exact version number you're using, E.g. NodeJS: 18.13.0 LTS
The future people in 20XX might run into issues when running the project as there can be a lot of things that might not work between major versions 🤣

Reply:
Fixed on this [commit](https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/30/commits/4006342fb8cd9bc627cb3a1cc80b9fb474d110b5)